src/components/Modal/index.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
```

Comment:
Actually instead of having this Modal component wrapping AttendeesModal and Confirmation Modal, can you just create 2 different components? I don't really like how the props are handled when they're combined